shared/events/edgeware/filters/labeler.ts

```diff
@@ -90,6 +90,16 @@ const labelEdgewareEvent: LabelerFilter = (
       };
     }
 
+    /**
+     * Offences Events
+     */
+    case SubstrateEventKind.Offence: {
+      const { offenceKind, opaqueTimeSlot, applied } = data;
+      return {
+        heading: 'Offence',
+        label: `There is an offence reported of the kind ${offenceKind} happened at the time slot ${opaqueTimeSlot} and applied ${applied}.`,
```

Comment:
We should rewrite this in active voice. Also, do we have a link URL? Or should the notification not direct the user anywhere?

Reply:
Nope. I couldn't formulate a URL from the data. 
Can I redirect to `{chain}/validators`?

Reply:
Yeah, that would be fine.